doc/source/reference/random/legacy.rst

```diff
@@ -121,3 +121,63 @@ Distributions
    ~RandomState.wald
    ~RandomState.weibull
    ~RandomState.zipf
+
+Toplevel `numpy.random` functions
+=================================
+Many of the RandomState methods above are exported as top-level `numpy.random`
```

Comment:
I'd also remove top-level here. And add a mention that using a `Generator` instance, as can be obtained from `default_rng`, should be preferred.

Reply:
fixing